### DISCUSSION TITLE
Is the development still live?

### DISCUSSION THREAD

**Discussion by ghost** [JMS55/sandbox#41] [Q&A] [answered]
Hello, I found this game recently and I thought it was cool and reminded me of an old mobile game,
is the game still being worked on?

**Comment by JMS55** [comment, marked as answer]
Hi, glad you enjoyed Sandbox!

Currently, I have no plans to add more content to Sandbox. I occasionally update dependencies and do maintenance on the project when I have time, including new packaged releases to Flathub, but gameplay wise Sandbox is effectively finished.

**Reply by ghost** [reply]
Ah alright, 
Good evening :).
